File: Iniciante/1009/1009.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){

    string name;
    double fixo, vendas, total;

    cin >> name >> fixo >> vendas;

    total = fixo+(vendas*0.15);

    cout << "TOTAL = R$ " << fixed << setprecision(2) << total << endl;

    return 0;
}

File: Iniciante/1009/1009.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner i = new Scanner(System.in);
        String name = i.nextLine();
        double fix_salary = i.nextDouble();
        double vendas = i.nextDouble();
        double bonus = vendas*0.15;
        System.out.printf("TOTAL = R$ %.2f\n", bonus+fix_salary);
    }   
}
